ui/src/ducks/inventory/index.js

```diff
@@ -1,17 +1,31 @@
+import axios from 'axios'
 import { createAction, handleActions } from 'redux-actions'
 
 const actions = {
   INVENTORY_GET_ALL: 'inventory/get_all',
+  INVENTORY_REFRESH: 'inventory/refresh',
+
 }
 
 export let defaultState = {
   all: []
 }
 
-export const findInventory = createAction(actions.INVENTORY_GET_ALL, () => {
-  //TODO
-})
+export const findInventory = createAction(actions.INVENTORY_GET_ALL, () => 
+  (dispatch, getState, config) => axios
+    .get(`${config.restAPIUrl}/inventory`)
+    .then((suc) => dispatch(refreshInventory(suc.data)))
+)
+
+export const refreshInventory = createAction(actions.INVENTORY_REFRESH, (payload) =>
+  (dispatcher, getState, config) =>
+     payload.sort((inventoryA, inventoryB) => inventoryA.name < inventoryB.name ? -1 : productA.name > productB.name ? 1 : 0)
```

Comment:
Although this works and even though you aren't comparing against two different fields like the article suggests, I would suggest switching out the sort function here for something like what this suggests (specifically the localeCompare): https://www.benmvp.com/blog/quick-way-sort-javascript-array-multiple-fields/